src/lsp_data.rs

```diff
@@ -219,6 +219,11 @@ pub fn completion_kind_from_match_type(m: racer::MatchType) -> CompletionItemKin
         | racer::MatchType::Static => CompletionItemKind::Variable,
         racer::MatchType::TypeParameter(_) => CompletionItemKind::TypeParameter,
         racer::MatchType::Builtin(_) => CompletionItemKind::Keyword,
+        racer::MatchType::UseAlias(m) => match m.mtype {
```

Comment:
@kngwyu do I understand that in this case, we have `use X as Y` and we match on `Y` but `m` here refers to `X` and we can derive the completion kind from that instead? (also nested/recursive case seems to be impossible?)

Reply:
I think it's impossible too but haven't tested it yet.